pocean/dsg/utils.py

```diff
@@ -72,6 +72,9 @@ def get_geographic_attributes(df, axes=None):
 
     if len(set(coords)) == 1:
         geoclass = Point
+        # The set is to workaround the fact tht pocean
+        # relied in a shapely<2 bug to pass a vector here instead of a point.
+        coords = set(coords)
```

Comment:
Not sure if you want to leave this as-is or to re-write it but it was relying on a bad behavior before. One cannot pass 4 coords to a Point class.

Reply:
From the Shapely 2.0 [release notes](https://shapely.readthedocs.io/en/stable/release/2.x.html#breaking-api-changes):

> The Point(..) constructor no longer accepts a sequence of coordinates consisting of more than one coordinate pair (previously, subsequent coordinates were ignored) (#1600).